### PR TITLE
Add "Differences between Personal Plan vs. Pro Plan"

### PR DESCRIPTION
**Pull request by dallasread**
This PR adds an article that discusses the differences between the Personal/Pro plans.

## 👓 Preview

image

### CODE REVIEW

**Comment by laetitiaeden** [comment]
we are going to change some of the text and maybe change the format how the comparaison 

pervs pro 
most important to showcase 

image

File: content/articles/personal-vs-pro.markdown [hunk 1]
```diff
@@ -28,44 +28,44 @@ This article describes the differences between the *Personal* plan and the *Prof
...
-**Personal:** On our personal plan, we offer Let's Encrypt certificates, but Wildcards are not included.
+**Personal:** Limited - We offer Let's Encrypt certificates on the Personal plan. Wildcards are not included.
 
-**Professional:** Yes! You'll be saving time and money with our Wildcard Let's Encrypt certificates.
+**Professional:** Yes - Save time and money with our Wildcard Let's Encrypt certificates.
 
 ## How fast can I get help from the experts?
 
-**Personal:** A watching-paint-dry 24 hours.
+**Personal:** Recieve an answer in 24 hours.
```
Comment (dallasread) [inline content/articles/personal-vs-pro.markdown]: `I` before `E` _except_ after `C`.

Comment (itsalyse) [reply content/articles/personal-vs-pro.markdown]: I already fixed it.

**Review by itsalyse — APPROVED** [review]

**Comment by dallasread** [comment]
@sbastn Any ideas to make this a little easier to read?

We started with a table that included all the text, but the content column is not enough pixels wide for it to look good.

**Comment by sbastn** [comment]
@dallasread ship it as is. We can monitor the visits to this page and think later on how to improve the text. Suggestions might also come from customer support. But this layout makes sense to me. You have the summary at the top followed by the details.

Is this page linked from anywhere else?

**Review by sbastn — APPROVED** [review]

**Comment by dallasread** [comment]
@sbastn It is not linked anywhere... yet. Perhaps coming up in an email campaign.

**Pull request merged**